app/javascript/components/Greeting.js

```diff
@@ -14,8 +15,10 @@ const Greeting = () => {
 
   return (
     <React.Fragment>
-      <p>{ message }</p>
-      <button onClick={() => dispatch(getGreeting())}>Fetch Greetings</button>
+      <div className="greeting">
+        <h1>{ message }</h1>
+        <button onClick={() => dispatch(getGreeting())}>Greet</button>
```

Comment:
- [ ] Right now, if I click the button to display the greeting, it shows me an error in the console about a type error, you may have missed something when setting your data. Here is a screenshot of the error:
![Captura de pantalla_2022-05-08_00-52-08](https://user-images.githubusercontent.com/59100477/167274452-71b34677-ffc9-40ff-b097-5936387411c6.png)
So please work on this issue.

Reply:
Hello @Lordkaito,

Thank you for the feedback. I don't get the error you are getting while running it on my local machine. Could you please have another look at the project? Below is a screenshot showing my logs.

![Screenshot from 2022-05-08 00-19-40](https://user-images.githubusercontent.com/49421407/167275080-d883be69-c82e-4a0a-bab6-34dfdb3e4ce3.png)